front/pages/super-user/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { GetServerSideProps, InferGetServerSidePropsType } from "next";
```

Comment:
Let’s just go with poke in the routes everywhere?

Reply:
Ok -- wasn't sure if we wanted that to only be a display name. Fine with it